libc-top-half/musl/src/misc/dl.c

```diff
@@ -0,0 +1,31 @@
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <dlfcn.h>
+
+#define EXPORT __attribute__((visibility("default"))) __attribute__((used))
+#define WEAK __attribute__((__weak__))
+
+static const char *error = 0;
+
+EXPORT WEAK int dlclose(void *library) {
+    error = "dlclose not implemented";
```

Comment:
I guess we should follow musl conventions here which are (for better or worse!) to use tabs for indentation, and to put the opening brace of the function on a line by istelf.